src/deps.txt

```diff
@@ -98,3 +98,6 @@ fedora-repos-ostree
 
 # For graphing manifest includes using `manifest_graph`
 python-anytree
+
+# For mkfs.erofs
+erofs-utils
```

Comment:
Minor/optional: this is already included in `vmdeps.txt` so it doesn't need to be here as well.